js/main.js

```diff
@@ -16,17 +16,137 @@ if (previousEntries) {
 $form.addEventListener('submit', function (e) {
```

Comment:
For some reason I can't comment on line 13, but you will want to be saving that into `data` instead of `data.entries` after you've completed my requested revision 👍